src/basic/terminal-util.h

```diff
@@ -11,19 +11,35 @@
 #include "time-util.h"
 
 /* Regular colors */
+#define ANSI_BLACK   "\x1B[0;30m"
 #define ANSI_RED     "\x1B[0;31m"
 #define ANSI_GREEN   "\x1B[0;32m"
 #define ANSI_YELLOW  "\x1B[0;33m"
 #define ANSI_BLUE    "\x1B[0;34m"
 #define ANSI_MAGENTA "\x1B[0;35m"
+#define ANSI_CYAN    "\x1B[0;36m"
+#define ANSI_WHITE   "\x1B[0;37m"
+
+#define ANSI_BRIGHT_BLACK   "\x1B[0;90m"
+#define ANSI_BRIGHT_RED     "\x1B[0;91m"
+#define ANSI_BRIGHT_GREEN   "\x1B[0;92m"
+#define ANSI_BRIGHT_YELLOW  "\x1B[0;93m"
+#define ANSI_BRIGHT_BLUE    "\x1B[0;94m"
+#define ANSI_BRIGHT_MAGENTA "\x1B[0;95m"
+#define ANSI_BRIGHT_CYAN    "\x1B[0;96m"
+#define ANSI_BRIGHT_WHITE   "\x1B[0;97m"
```

Comment:
we never used the 9x ANSI sequences so far, why introduce them now? (aixterm stuff...)

I mean it makes sense to complete the set of colors we use already, but adding new sets of colors for testing?

Reply:
You're right, I'll just remove that part.

Reply:
Hmm, on second thought, let's keep it. Each of the three colors (normal, bold, highlight) is rendered differently. So I think it makes sense to display all of them, so that when we again need to pick a color for something, we can figure out how the different options look.

Reply:
maybe add a comment though...

i wonder how universally accepted these are? does xterm + gnome-terminal + konsole + linux console all support them? (i guess these are the big four terminals we care about)

Reply:
I tested on gnome-terminal, konsole, urxvt, xterm, and linux console, and they all support this.